gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dsubmenubutton_ttt2.lua

```diff
@@ -123,7 +123,7 @@ end
 ---
 -- @ignore
 function PANEL:PerformLayout()
-	DLabel.PerformLayout(self)
+	DLabel.PerformLayout(self) -- where does DLabel come from?!?!
```

Comment:
Maybe this is actually correct, but most of the GUI stuff just goes right above my head :/

Reply:
I spotted some elements like this in the base gmod code. I guess this is either a bug or something "localized" as a global

Reply:
You are right. Maybe this is correct then. I'll take my chances and leave this as is.